MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import MapboxDirections
+
+/**
+ A `NavigationRouteOptions` object is a structure that specifies the criteria for results returned by the Mapbox Directions API.
+ 
+ `NavigationRouteOptions` is a subclass of `RouteOptions` that has been optimized for navigation. Pass an instance of this class into the `Directions.calculate(_:completionHandler:)` method.
+ */
+@objc(MBNavigationRouteOptions)
+open class NavigationRouteOptions: RouteOptions {
+    
+    /**
+     Initializes a navigation route options object for routes between the given waypoints and an optional profile identifier optimized for navigation.
+     
+     - parameter waypoints: An array of `Waypoint` objects representing locations that the route should visit in chronological order. The array should contain at least two waypoints (the source and destination) and at most 25 waypoints. (Some profiles, such as `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic`, [may have lower limits](https://www.mapbox.com/api-documentation/#directions).)
+     - parameter profileIdentifier: A string specifying the primary mode of transportation for the routes. This parameter, if set, should be set to `MBDirectionsProfileIdentifierAutomobile`, `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic`, `MBDirectionsProfileIdentifierCycling`, or `MBDirectionsProfileIdentifierWalking`. `MBDirectionsProfileIdentifierAutomobile` is used by default.
```

Comment:
I’m a little uncomfortable with mirroring all the RouteOptions documentation here. Could we get away with a `- see:` command? We might need to beef up [this line](https://github.com/mapbox/mapbox-navigation-ios/blob/5f3856376eaead146a58688bbed40df842b54e9d/scripts/document.sh#L50) that inserts links from the MapboxNavigation docset to the MapboxDirections docset.

Reply:
Sorry, that should be [`- SeeAlso:`](https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_markup_formatting_ref/SeeAlso.html).